### PR TITLE
git commit -m "adding dearborn and gu jobs thrashers”

### PR DESCRIPTION
**Pull request by LisaVillani**
hopefully adding a couple of thrashers!

### CODE REVIEW

File: embeds/docs-dearborn-michigan/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
... 3.02-6 3.02zm-6.2-6.83c-2.5.5-5.2 1.5-5.2 4.3v.2c-2.8-2.9-4.6-6.8-4.6-11.1 0-6.4 3.8-11.9 9.2-14.4-1.9 1.3-2.9 3.2-2.9 5.6 0 2.8 1.4 4.9 3.6 6.1l.1.1c-1.3.8-3.8 2.8-3.8 5.1 0 1.7 1.1 3.3 3.4 3.8zm6-20.2c1.6 0 2.9.6 2.9 5 0 4.3-1.4 4.7-2.9 4.7-1.6 0-2.9-.4-2.9-4.8s1.3-5 2.8-5zm34.8 12.5c0-1.3.5-2.3 1.3-2.9L14 22.4 11.9 24v46.3l2.1 1.44 53-21.7c-1.8-.28-2.75-1.6-2.75-3.5z" opacity=".8"/><path d="M51.94 46.58c0 1.66.42 2.38 1.3 2.38.35 0 .54-.06.7-.12v-4.56c-.17-.13-.37-.24-.76-.24-.7 0-1.24.64-1.24 2.54m8.6-2.58c-.75 0-1.12.4-1.12 2.62 0 2.23.37 2.6 1.12 2.6.75 0 1.1-.37 1.1-2.6 0-2.22-.35-2.6-1.1-2.6" opacity=".8"/><path d="M60.54 49.23c.7 0 1.06-.35 1.1-2.3l-2.2-.92c-.02.2-.02.4-.02.7 0 2.3.37 2.6 1.12 2.6m-8.6-2.6c0 1.7.42 2.4 1.3 2.4.35 0 .54 0 .7-.1v-4.6c-.17-.1-.37-.2-.76-.2-.7 0-1.24.7-1.24 2.6" opacity=".8"/><path d="M67.82 50.16c-1.7 0-2.77-.77-3.23-2.03l-.8-.3c-.4 1.52-1.7 2.34-3.3 2.34-2.1 0-3.5-1.26-3.5-3.55 0-.55.1-1.05.2-1.48l-1.2-.47v4.2l.7.4v.57l-2.7.3v-.7h-.1c-.3.36-.9.73-1.7.73-1.3 0-2.7-.78-2.7-3.37 0-2.5 1.3-3.52 2.8-3.67l-6.8-2.8-1.6 1.1c.9 1.96 1.33 4.1 1.33 6.38 0 2.3-.5 4.5-1.38 6.5v22.4l1.6 1.3 43.3-17.8v-2L69 50c-.34.1-.75.2-1.18.2" opacity=".8"/></g></svg>
```
Comment (sndrs) [inline embeds/docs-dearborn-michigan/index.html]: you [need to remove](http://stuntbox.com/blog/2013/06/bulletproof-inline-svg/) the `xmlns` attribute here

File: embeds/docs-dearborn-michigan/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+</div>
+
+<div class='foreground'>
+    <div class='title'><span class="spanstyle">Dearborn,</span> Michigan</div>
+    <div class='strapline'>
+        <span>The divided city debating what it means to be an American today</span>
+    </div>
+    <div class='play'>
+        <svg viewBox="13 13 26 21" xmlns="http://www.w3.org/2000/svg"><path fill="#333" fill-rule="evenodd" d="M38.5 24.08V23L14.4 13l-.9.7v19.68l.9.62"/></svg>
```
Comment (sndrs) [inline embeds/docs-dearborn-michigan/index.html]: `xmlns` again (and any others)

**Comment by akash1810** [comment]
@sndrs 

> on a quick test, embeds/docs-dearborn-michigan/_source/background-image.png looks like it could be a 20kb jpg instead?

We could use something like [bundlesize](https://github.com/siddharthkp/bundlesize) to enforce this in CI? Granted, its probably a misuse of bundlesize...

**Comment by LisaVillani** [comment]
Sometimes I think it's still 1999!

**Comment by sndrs** [comment]
@akash1810 that's a great idea! check the size of the json?

**Pull request merged**